shared/constants/types/chat2/index.tsx

```diff
@@ -130,9 +131,10 @@ export type AudioRecordingInfo = {
   status: AudioRecordingStatus
   outboxID: Buffer
   path: string
+  recordEnd?: number
   recordStart: number
   isLocked: boolean
-  amps: Array<number>
+  amps?: AmpTracker
```

Comment:
`AmpTracker` is a class that mutates itself. Should probably turn it into an object and move respective operations into the reducer?

Reply:
I don't want any of this stuff to go through Redux

Reply:
Maybe just store it separately then? Kind of weird to put something not mutated by reducer inside the store.